src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerGetAdminMediasByIdIntTest.java

```diff
@@ -271,6 +273,65 @@ void shouldDenyAccess(SecurityRoleEnum role) throws Exception {
                                     """, jsonString, JSONCompareMode.STRICT);
     }
 
+    @Test
+    void getMediaById_shouldReturnCaseWithCasIdAndActualHearing_whenMediaHasLinkedCase() throws Exception {
+        // Given
+        given.anAuthenticatedUserWithGlobalAccessAndRole(SecurityRoleEnum.SUPER_USER);
+        UserAccountEntity userAccountEntity = databaseStub.getUserAccountRepository().findAll().stream()
+            .findFirst()
+            .orElseThrow();
+
+        // Create hearing and mark it as actual
+        var hearingEntity = databaseStub.createHearing(COURTHOUSE_NAME, COURTROOM_NAME, CASE_NUMBER, HEARING_START_AT.toLocalDateTime());
+        hearingEntity.setHearingIsActual(true);
+        databaseStub.getHearingRepository().save(hearingEntity);
+
+        // Create media and link it to hearing
+        var mediaEntity = createAndSaveMediaEntity(hearingEntity, userAccountEntity, false);
+
+        // Create media linked case with court case (AC1)
```

Comment:
Having AC1 in the comment doesn't really mean anything to a dev in the future. Probably worth just removing. Same comment further down.

Reply:
agree - have removed